src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -1675,11 +1677,11 @@ public sealed partial class MetadataReaderProvider : System.IDisposable
         public void Dispose() { }
         public unsafe static System.Reflection.Metadata.MetadataReaderProvider FromMetadataImage(byte* start, int size) { throw null; }
         public static System.Reflection.Metadata.MetadataReaderProvider FromMetadataImage(System.Collections.Immutable.ImmutableArray<byte> image) { throw null; }
-        public static System.Reflection.Metadata.MetadataReaderProvider FromMetadataStream(System.IO.Stream stream, System.Reflection.Metadata.MetadataStreamOptions options = (System.Reflection.Metadata.MetadataStreamOptions)(0), int size = 0) { throw null; }
+        public static System.Reflection.Metadata.MetadataReaderProvider FromMetadataStream(System.IO.Stream stream, System.Reflection.Metadata.MetadataStreamOptions options = System.Reflection.Metadata.MetadataStreamOptions.Default, int size = 0) { throw null; }
         public unsafe static System.Reflection.Metadata.MetadataReaderProvider FromPortablePdbImage(byte* start, int size) { throw null; }
         public static System.Reflection.Metadata.MetadataReaderProvider FromPortablePdbImage(System.Collections.Immutable.ImmutableArray<byte> image) { throw null; }
-        public static System.Reflection.Metadata.MetadataReaderProvider FromPortablePdbStream(System.IO.Stream stream, System.Reflection.Metadata.MetadataStreamOptions options = (System.Reflection.Metadata.MetadataStreamOptions)(0), int size = 0) { throw null; }
-        public System.Reflection.Metadata.MetadataReader GetMetadataReader(System.Reflection.Metadata.MetadataReaderOptions options = (System.Reflection.Metadata.MetadataReaderOptions)(1), System.Reflection.Metadata.MetadataStringDecoder utf8Decoder = null) { throw null; }
+        public static System.Reflection.Metadata.MetadataReaderProvider FromPortablePdbStream(System.IO.Stream stream, System.Reflection.Metadata.MetadataStreamOptions options = System.Reflection.Metadata.MetadataStreamOptions.Default, int size = 0) { throw null; }
+        public System.Reflection.Metadata.MetadataReader GetMetadataReader(System.Reflection.Metadata.MetadataReaderOptions options = System.Reflection.Metadata.MetadataReaderOptions.Default, System.Reflection.Metadata.MetadataStringDecoder utf8Decoder = null) { throw null; }
```

Comment:
The documentation says `Metadata.MetadataReaderOptions.ApplyWindowsRuntimeProjections` is on by default. Should this be updated accordingly?

https://github.com/dotnet/corefx/blob/8a9dd52ae5bab85ec3bd1d3cf73ba165668332f0/src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReaderOptions.cs#L22-L25
https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReaderProvider.cs#L246

Reply:
I'm not sure I understand the question.  `Default` and `ApplyWindowsRuntimeProjections` are both valued as 1.

Reply:
Oh, lol, you are right. I missed that in the sources `Default` is set to `ApplyWindowsRuntimeProjections`. It was odd to see both `(System.Reflection.Metadata.MetadataStreamOptions)(0)` and (System.Reflection.Metadata.MetadataStreamOptions)(1)` point to the same enum value. They were originally set to different enum values in the ref but the source has both set to `.Default`.

Reply:
`(System.Reflection.Metadata.MetadataStreamOptions)(0)` should have gone to `None`.  If it went to `Default` there's a change being introduced.

Reply:
Here's the confusion: 
StreamOptions != ReaderOptions

(StreamOptions)(0) is `Default`, not `None`:
https://github.com/dotnet/corefx/blob/26b932e3c56702382d13ece010b300d3181d2891/src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataStreamOptions.cs#L13

(ReaderOptions)(0) is `None`:
https://github.com/dotnet/corefx/blob/26b932e3c56702382d13ece010b300d3181d2891/src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReaderOptions.cs#L13